### ISSUE TITLE
reminders

### ISSUE THREAD

**Issue by bbelderbos** [CodeConnoisseur74/pdi-to-do-app#7]

**Comment by bbelderbos** [comment]
- a process that runs once a day and checks for events that are < 24 hours due (dependency = #5) and then notifies the user, maybe by email or some kind of notification
- in former case you need to have the user set their email somewhere (similar to initial git config)

**Comment by CodeConnoisseur74** [comment]
https://chat.openai.com/share/eb7f9271-f429-4794-a9c9-f93338950da5

**Comment by bbelderbos** [comment]
This thread is related to recurring events, I would not go that route now, let's do reminders instead. Can you brainstorm that feature here instead? Thanks.

**Comment by CodeConnoisseur74** [comment]
@bbelderbos Yes, I realized after I opened it that it had to do with recurring events. I'll brainstorm in here about how to go about it.

**Comment by CodeConnoisseur74** [comment]
@bbelderbos Yeah, I see that Emily has done a to do app as well. Perhaps, I should just do another one and probably do better at it actually. Plus, it would add a little more diversity to the program on the outside. 

It may make potential clients think that a To Do App is the only option. 

I would have chosen another Capstone Project if I knew this one was the only one I was doing. I'm not really feeling it if you can tell. Plus I feel like an  *sshole with severe impostor syndrome. 

The weather project was the one I was hoping to do if I could pick one. I know it's super late in the game. However, I can certainly pull it off and I can use an API for it.

**Comment by bbelderbos** [comment]
Sure, good point, and you can still do it. 

Please shoot me a PR with initial iteration of the app Kajabi page's requirements, any questions upfront how to go about this?

**Comment by CodeConnoisseur74** [comment]
@bbelderbos Awesome and this is still a great repo and I plan on continuing it for sure. I could totally do this one though with an API. 

I'm super productive today and just wanted to get back into the game. The ability to have separate  google windows open, one for Pybites and one for my personal is amazing, along with the organizational app with workflows, providing two workspaces, one for what I do with Pybites and the other for personal, studies, etc, is a game changer!
